stock_inventory_discrepancy/models/stock_location.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# pylint: disable=import-error,protected-access,too-few-public-methods
+# Copyright 2017 Eficent Business and IT Consulting Services S.L.
+#   (http://www.eficent.com)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+"""stock_location"""
```

Comment:
What's this ? (and in other files)

Reply:
Thank you for the review. About the code, which one? 'pylint: disable...'? i add this because of the plugin in my script writter (atom) that automatically check the pylint error always mark 'from odoo import ...' as import-error. That is irritating. But i forget to delete it before push it. I'm sorry for that one and i'll delete it in the next push.

Reply:
@andreasdsp No, it was just the """stock_location""". But that's detail